2_process.R

```diff
@@ -38,5 +38,15 @@ p2_targets_list <- list(
      {
        sites_tbl <- read_csv(p2_site_list_csv)
        get_site_flowlines(p1_reaches_sf, sites_tbl, sites_crs = 4269, max_matches = 1, search_radius = 0.1)
-     })
+     }),
+  
+  # Aggregate instantaneous DO data to daily min/mean/max
```

Comment:
Do you mind editing the descriptions here to replace "DO" references with "SC" (and on line 48)?

Reply:
Haha good catch

Reply:
```suggestion
  # Aggregate instantaneous SC data to daily min/mean/max
```